src/components/ArticleList.jsx

```diff
@@ -0,0 +1,29 @@
+import fetchArticles from "../api-utils";
+import { useState, useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    fetchArticles().then((articleData) => {
+      setArticles(articleData);
+    });
+  }, []);
+
+  return (
+    <div>
+      <ul>
+        {articles.map((article) => {
+          return (
+            <li key={article.article_id}>
```

Comment:
No need for the wrapping `li` here. You can return the `<ArticleCard key={article.article_id} ...` with the key directly on it.